<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>10000</width>
    <height>10000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiLight SemiConde</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>交通换乘系统(支持北京上海)，祝您出行愉快!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QLabel" name="EndLine">
         <property name="text">
          <string>终点线路</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QComboBox" name="EndLineQComboBox"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="StationLessRadioButton">
            <property name="text">
             <string>站点少</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="LookUpPushButton">
            <property name="text">
             <string>查询线路</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="TransferLessRadioButton">
            <property name="text">
             <string>换乘少</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QTextBrowser" name="DisplayTextBrowser">
            <property name="font">
             <font>
              <family>Bahnschrift SemiBold SemiConden</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bahnschrift SemiBold SemiConden'; font-size:14pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-size:12pt; font-weight:400;&quot;&gt;推荐线路：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="StartLineQComboBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="StartLine">
         <property name="text">
          <string>起点线路</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="StartStationQComboBox"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="EndStationQComboBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="StartStasion">
         <property name="text">
          <string>起点站</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="EndStation">
         <property name="text">
          <string>终点站</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MyQGraphicsView" name="QGraphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>10000</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="MenuOperation">
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>视图操作入口</string>
    </property>
    <addaction name="ActionEnlargeMap"/>
    <addaction name="separator"/>
    <addaction name="ActionZoomInMap"/>
    <addaction name="ActionDisplayAllMapBJ"/>
    <addaction name="ActionDisplayAllMapSH"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>北京</string>
    </property>
    <addaction name="ActionOnlySubwayBJ"/>
    <addaction name="ActionOnlyBusBJ"/>
    <addaction name="BJ"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>上海</string>
    </property>
    <addaction name="ActionOnlySubwaySH"/>
    <addaction name="ActionOnlyBusSH"/>
    <addaction name="SH"/>
   </widget>
   <addaction name="MenuOperation"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="ActionEnlargeMap">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/enlarge.png</normaloff>:/images/enlarge.png</iconset>
   </property>
   <property name="text">
    <string>放大视图</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ActionZoomInMap">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/shrink.png</normaloff>:/images/shrink.png</iconset>
   </property>
   <property name="text">
    <string>缩小视图</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="ActionDisplayAllMapBJ">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/map.png</normaloff>:/images/map.png</iconset>
   </property>
   <property name="text">
    <string>展示完整地图(北京）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="ActionDisplayAllMapSH">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>展示完整地图(上海)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="ActionOnlySubwayBJ">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/subway.png</normaloff>:/images/subway.png</iconset>
   </property>
   <property name="text">
    <string>仅地铁</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="ActionOnlyBusBJ">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>仅公交</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="BJ">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/text.png</normaloff>:/images/text.png</iconset>
   </property>
   <property name="text">
    <string>公交+地铁</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="ActionOnlySubwaySH">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/subway.png</normaloff>:/images/subway.png</iconset>
   </property>
   <property name="text">
    <string>仅地铁</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="ActionOnlyBusSH">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>仅公交</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="SH">
   <property name="icon">
    <iconset resource="resource/resource.qrc">
     <normaloff>:/images/all.png</normaloff>:/images/all.png</iconset>
   </property>
   <property name="text">
    <string>公交+地铁</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>myqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource/resource.qrc"/>
  <include location="resource/resource.qrc"/>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
